service-level-indicators/spark-retrieval-success-rate.md

```diff
@@ -0,0 +1,612 @@
+    - Created during [Retrieval Checks](#retrieval-checks) and [Reporting Measurements to Spark-API](#reporting-measurements-to-spark-api)
+    - Created by [Checker / Checker Node](#checker--checker-node)
+    - Spark-API receives measurements and store them in the publishing queue.
+    - Spark-Publish uploads them to Storacha.
+    - Spark-Evaluate downloads the measurements and process them as part of the round evaluation.
+    - Consumed by Spark-API
+
+### Committee Retrieval Task Measurements
+
+- `(roundId, providerId, payloadCID, retrievalTaskMeasurements)`
+    - This is the set of ALL [Retrieval Task Measurement](#retrieval-task-measurement)s submitted by the network for a [Committee](#committee), including invalid/fraudulent measurements.
+    - Created at the end of [Reporting Measurements to Spark-API](#reporting-measurements-to-spark-api)
+    - Created by multiple by [Checker / Checker Node](#checker--checker-node)s
+    - Consumed by Spark-Evaluate
+
+### Accepted Retrieval Task Measurement
+
+- `(roundId, providerId, payloadCID, checkerId, retrievalResult, retrtievalResultCode, clients)`
+    - A [Retrieval Task Measurement](#retrieval-task-measurement) that passes the fraud detection checks during [Evaluating Measurements with Spark-Evaluate](#evaluating-measurements-with-spark-evaluate)
+    - Created during [Evaluating Measurements with Spark-Evaluate](#evaluating-measurements-with-spark-evaluate)
+    - Created by Spark-Evaluate
+    - Consumed by Spark-Evaluate
+
+### Committee Accepted Retrieval Task Measurements
+
+- `(roundId, providerId, payloadCID, checkerId, retrievalResult, retrtievalResultCode, clients)`
+    - The [Accepted Retrieval Task Measurement](#accepted-retrieval-task-measurement)s for a [Committee](#committee).
+    - Created during [Evaluating Measurements with Spark-Evaluate](#evaluating-measurements-with-spark-evaluate)s.  This is what is stored off chain, with a pointer on chain.
+    - Created by Spark-Evaluate
+    - Consumed by Spark-Evaluate
+
+### Provider Retrieval Result Stats
+
+```jsx
+{
+  "date": "<YYYY-MM-DD>",
+  "meta": {
+    "rounds": [
+      {
+        "index": "<roundIndex>",
+        "contractAddress": "<contractAddress>",
+        // This is the "Retrieval Task List" for the round.
+        "details": { "/": "<cid>" }
+        // These are the "Retrieval Task Measurements" that are uploaded as part of "Reporting Measurements to Spark-API".  
+        "measurementBatches": [{ "/": "<cid>" }, ...],
+        "sparkEvaluateVersion": {
+          "gitCommit": "<hash>",
+        },
+      },
+      ...
+    ]
+  },
+  "providerRetrievalResultStats": [
+    {
+      "providerId": "<providerId>",
+      "successful": <n>,
+      "total": <n>
+    },
+    ...
+  ]
+}
+```
+
+- Created by X on a daily basis as part of Y
+- Consumed by anyone wanting to analyze or display SP retrievability performance.
+
+## Callouts/Concerns with this SLI
```

Comment:
TODO: add one that checker traffic is pretty easy to differentiate from real traffic (given checker traffic only requests the `payloadCid`).  Basically a SP could just return to payloadCid requests, have 100% Spark RSR, but actually not enabling guaranteed retrieval.

Reply:
Done in https://github.com/filecoin-project/service-classes/pull/7. Resolving.